src/main/java/potong/command/MarkCommand.java

```diff
@@ -0,0 +1,46 @@
+package potong.command;
+
+import potong.exceptions.PotongException;
+
+import potong.Storage;
+import potong.TaskList;
+import potong.Ui;
+
+/**
+ * Represents the command to mark or unmark a task.
+ */
+public class MarkCommand extends Command {
+
+    private boolean toMark;
```

Comment:
Should sound more like a boolean?
```suggestion
    private boolean isMarked;
```